prow/cluster/build/buildcache.yaml

```diff
@@ -18,7 +17,6 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   labels:
-    app.kubernetes.io/part-of: prow-build
```

Comment:
if there are no other labels left, doesn't this fail due to being invalid yaml?

Reply:
Oh, I think you're right. Let me fix that.

Reply:
Thanks Jacob, I learned about multi-line sed and fixed things up. I've skimmed through and everything looks right to me now.